ScoutSuite/providers/aws/rules/findings/iam-root-account-used-recently.json

```diff
@@ -1,7 +1,15 @@
 {
     "description": "Root account used recently",
-    "path": "iam.credential_reports.id",
+    "rationale": "<b>Description:</b><br><br>The use of the root account should be avoided.",
```

Comment:
Remove `<b>Description:</b><br><br>`

Reply:
This is fixed and will be available in the next commit.